web/web/src/app/rootLayout/AppBar.js

```diff
@@ -64,6 +66,7 @@ const AppBar = () => {
     const metalakeItems = store.metalakes.map(i => i.name)
     setMetalakes(metalakeItems)
     // eslint-disable-next-line react-hooks/exhaustive-deps
+    axios.get('/')
```

Comment:
What is this request for?

Reply:
For testing axios function, I forget to remove it.